chia/full_node/mempool_manager.py

```diff
@@ -294,6 +295,11 @@ async def add_spendbundle(
         for name in removal_names:
             removal_record = await self.coin_store.get_coin_record(name)
             if removal_record is None and name not in additions_dict:
+                log.error(
+                    f"MempoolInclusionStatus.FAILED, Err.UNKNOWN_UNSPENT:\n"
```

Comment:
This also doesn't need the f in principle. Not sure if it's a convenience measure at this point.